src/components/data-management/ProjectDetails.jsx

```diff
@@ -452,7 +452,7 @@ const ProjectDetails = ({ width, height }) => {
   ];
 
   useEffect(() => {
-    if (projects.ids.length === 0 || samples.ids.length === 0) {
+    if (projects.ids.length === 0 || !activeProject || !samples.ids.includes(activeProject.samples[0])) {
```

Comment:
`samples.ids.includes(activeProject.samples[0])` can error out if `samples` is null or `activeProject` is null

Reply:
That's why there is the condition for the `!activeProject` before that, `samples.ids` and each project's samples are never null they are just empty arrays so the third condition won't break too.

Reply:
ok, sounds good!